lte/gateway/c/oai/tasks/nas/nas_procedures.c

```diff
@@ -656,7 +656,7 @@ nas_sr_proc_t* nas_new_service_request_procedure(
     return NULL;
   }
   emm_context->emm_procedures->emm_con_mngt_proc =
-      calloc(1, sizeof(nas_sr_proc_t));
```

Comment:
The Service Request Procedure `nas_sr_proc_t` (https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/nas_procedures.h#L267) encapsulates `nas_emm_con_mngt_proc_t` and this EMM procedure needs the `nas_sr_proc_t`.

Reply:
Thanks for the clarification @ssanadhya !